examples/blueprint-construct/index.ts

```diff
@@ -29,17 +29,17 @@ export default class BlueprintConstruct extends cdk.Construct {
         // const platformTeam = new team.TeamPlatform(account)
 
         // Teams for the cluster.
-        const teams: Array<ssp.Team> = [
+        const teams: Array<blueprints.Team> = [
             new team.TeamTroi,
             new team.TeamRiker(scope, teamManifestDirList[1]),
             new team.TeamBurnham(scope, teamManifestDirList[0]),
             new team.TeamPlatform(process.env.CDK_DEFAULT_ACCOUNT!)
         ];
-        const prodBootstrapArgo = new ssp.addons.ArgoCDAddOn({
+        const prodBootstrapArgo = new blueprints.addons.ArgoCDAddOn({
             // TODO: enabling this cause stack deletion failure, known issue:
-            // https://github.com/aws-quickstart/ssp-amazon-eks/blob/main/docs/addons/argo-cd.md#known-issues
+            // https://github.com/aws-quickstart/cdk-eks-blueprints/blob/main/docs/addons/argo-cd.md#known-issues
             // bootstrapRepo: {
-            //      repoUrl: 'https://github.com/aws-samples/ssp-eks-workloads.git',
+            //      repoUrl: 'https://github.com/aws-samples/blueprints-eks-workloads.git',
```

Comment:
can we drop hte commented code?

Reply:
it is handy for local testing with bootstrapping. This is not part of the npm module. There is a TODO to address it (e.g. uncomment for e2e tests)